orion-server/src/main/java/com/pinterest/orion/core/memq/MemqCluster.java

```diff
@@ -65,12 +67,15 @@ public MemqCluster(String id,
         costCalculator);
   }
 
+  @SuppressWarnings("unchecked")
   @Override
   protected void bootstrapClusterInfo(Map<String, Object> config) throws PluginConfigurationException {
     this.config = config;
     setAttribute(ZK_CONNECTION_STRING, config.get(ZK_CONNECTION_STRING));
     setAttribute(SERVERSET_PATH, config.get(SERVERSET_PATH));
     setAttribute(CLUSTER_INFO_DIR, config.get(CLUSTER_INFO_DIR));
+    notificationClusterConfig = (Map<String, String>) config
+        .get(MemqTopicCreationAction.NOTIFICATION_CLUSTER_CONFIG);
```

Comment:
Should we move this constant `MemqTopicCreationAction.NOTIFICATION_CLUSTER_CONFIG` from the action to the cluster instead? It's seems to used by multiple components.

Reply:
Fixed